docs/contributing/stacks.md

```diff
@@ -65,6 +65,44 @@ git push -u origin master
 
 ## Configuring Travis
 
+Next, link your GitHub project to Travis CI to build your Docker image whenever you or someone else submits a pull request.
+
+1. Visit [https://docs.travis-ci.com/user/getting-started/#To-get-started-with-Travis-CI](https://docs.travis-ci.com/user/getting-started/#To-get-started-with-Travis-CI)
+2. Follow the instructions to add the Travis CI application to your GitHub account.
+3. Visit [https://travis-ci.org](https://travis-ci.org).
+4. Click the + symbol on the left side.
+5. Locate your project repository either in your primary user account or in one of the organizations to which you belong.
+6. Click the toggle to enable builds for the project repository.
+7. Click the *Settings* button for that repository.
```

Comment:
Format maybe with `code case` here

Reply:
Going with bold for consistency with the other change above.